addons/l10n_fr_certification/models/res_company.py

```diff
@@ -2,41 +2,65 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from openerp import fields, models, api
+from openerp.exceptions import UserError
+from openerp.tools.translate import _
+
+UNALTERABLE_COUNTRIES = ['FR', 'MF', 'MQ', 'NC', 'PF', 'RE', 'GF', 'GP', 'TF']
+
 
 class ResCompany(models.Model):
     _inherit = 'res.company'
 
+    # To do in master : refactor to set sequences more generic
+
     l10n_fr_secure_sequence_id = fields.Many2one('ir.sequence', 'Sequence to use to ensure the securisation of data', readonly=True)
+    l10n_fr_closure_sequence_id = fields.Many2one('ir.sequence')
```

Comment:
Hi, to be more understandable, could you rename your variables into : 
- l10n_fr_secure_sequence_id -> l10n_fr_account_move_sequence_id (with old_name, and updating version number)
- l10n_fr_closure_sequence_id -> l10n_fr_pos_order_sequence_id 

More readable / understandable for developers, and it will help, if in some times, other objects are certified. (picking, purchase - sale orders, ....)
thanks.

Reply:
@legalsylvain 
i can't change the secure sequence name on account_move (this field exists in stable now)
for l10n_fr_closure_sequence_id, it is not necessarily linked to pos orders, but to clures objects only, so no need to change it as you propose